neuralprophet/plot_plotly.py

```diff
@@ -0,0 +1,646 @@
+                "comp_name": "future_regressors_additive",
+            }
+        )
+    if "future_regressors_multiplicative" in fcst.columns:
+        components.append(
+            {
+                "plot_name": "Multiplicative Future Regressors",
+                "comp_name": "future_regressors_multiplicative",
+                "multiplicative": True,
+            }
+        )
+    if residuals:
+        if forecast_in_focus is None and m.n_forecasts > 1:
+            if fcst["residual1"].count() > 0:
+                components.append(
+                    {
+                        "plot_name": "Residuals",
+                        "comp_name": "residual",
+                        "num_overplot": m.n_forecasts,
+                        "bar": True,
+                    }
+                )
+        else:
+            ahead = 1 if forecast_in_focus is None else forecast_in_focus
+            if fcst["residual{}".format(ahead)].count() > 0:
+                components.append(
+                    {
+                        "plot_name": "Residuals ({})-ahead".format(ahead),
+                        "comp_name": "residual{}".format(ahead),
+                        "bar": True,
+                    }
+                )
+
+    npanel = len(components)
+    figsize = figsize if figsize else (10, 3 * npanel)
+
+    # Create Plotly subplot figure and add the components to it
+    fig = make_subplots(npanel, cols=1, print_grid=False)
+    fig["layout"].update(go.Layout(showlegend=False, width=figsize[0], height=figsize[1] * npanel))
+
+    multiplicative_axes = []
+    for i, comp in enumerate(components):
+        name = comp["plot_name"].lower()
+        ploty_trace = None
+
+        if (
+            name in ["trend"]
+            or ("residuals" in name and "ahead" in name)
+            or ("ar" in name and "ahead" in name)
+            or ("lagged_regressor" in name and "ahead" in name)
+        ):
+            trace_object = get_forecast_component_plotly_props(m, fcst=fcst, **comp)
+
+        elif "event" in name or "future regressor" in name:
+            trace_object = get_forecast_component_plotly_props(m, fcst=fcst, **comp)
+
+        elif "season" in name:
+            if m.season_config.mode == "multiplicative":
+                comp.update({"multiplicative": True})
+            if one_period_per_season:
+                comp_name = comp["comp_name"]
+                print(comp_name)
+                trace_object = get_seasonality_plotly_props(m, fcst, **comp)
+            else:
+                comp_name = f"season_{comp['comp_name']}"
+                trace_object = get_forecast_component_plotly_props(
+                    m, fcst=fcst, comp_name=comp_name, plot_name=comp["plot_name"]
+                )
+
+        elif "auto-regression" in name or "lagged regressor" in name or "residuals" in name:
+            trace_object = get_multiforecast_component_plotly_props(fcst=fcst, **comp)
```

Comment:
```suggestion
            trace_object = get_multiforecast_component_plotly_props(fcst=fcst, **comp)
            fig.update_layout(barmode='overlay')
```
Set the barchart mode as overlay